specification/cognitiveservices/OpenAI.Inference/client.tsp

```diff
@@ -5,15 +5,41 @@ using Azure.ClientGenerator.Core;
 
 // Azure-specific long-running operations should be treated as implementation details that are wrapped into
 // appropriately merged public surface.
-@@internal(Azure.OpenAI.beginAzureBatchImageGeneration);
-@@internal(Azure.OpenAI.getAzureBatchImageGenerationOperationStatus);
+@@access(Azure.OpenAI.beginAzureBatchImageGeneration, Access.internal);
+@@access(Azure.OpenAI.getAzureBatchImageGenerationOperationStatus, Access.internal);
 
 // Azure-specific Chat Completions with extensions should be handled by clients as a conditional selection within the
 // shared Chat Completions route, with the selection gated by the presence or non-presence of additional child
 // configuration options on the request payload options model.
-@@internal(Azure.OpenAI.getChatCompletionsWithAzureExtensions);
+@@access(Azure.OpenAI.getChatCompletionsWithAzureExtensions, Access.internal);
 
 // Some models from routes with suppressed visibility are still desired for custom public surface.
-@@include(Azure.OpenAI.ImageGenerationOptions);
-@@include(Azure.OpenAI.ImageLocation);
-@@include(Azure.OpenAI.ImageGenerations);
+@@access(Azure.OpenAI.ImageGenerationOptions, Access.public);
```

Comment:
ImageGenerationResponseFormat should be public.
`: "ImageGenerationResponseFormat" is in an implementation package and should not be a return type for public API. Alternatively, it can be removed from the implementation package and made public API, after appropriate API review. [NoImplInPublicAPI]`

I tried to use 
`
@@access(Azure.OpenAI.ImageGenerationResponseFormat, Access.public);
`
but it doesn't make ImageGenerationResponseFormat to be a public model.

Reply:
That is a bit strange. I believe I've done customization to hide `ImageGenerationResponseFormat` as package private (commit https://github.com/Azure/azure-sdk-for-java/pull/37179/commits/1b96296c47f1a21050c8919176a852afe6bd92f6)

And "Build Analyze" CI is pass.
https://github.com/Azure/azure-sdk-for-java/pull/37179/checks?check_run_id=17670608007

Reply:
`@@access(Azure.OpenAI.ImageGenerationResponseFormat, Access.public);` does not work because I've added it to tspconfig.yaml to move it to `implementation.models`.
```
    custom-types: "ImageGenerationResponseFormat"
```

(So it works for typespec, just we had another Java configure to move it)

Reply:
Yeah. `ImageGenerationResponseFormat` is probably not needed.

I heard from TypeSpec meeting that we will remove the LRO operation of Image Generation in the coming service API version 2023-12-01-preview. It might be best to wait for a bit to get it cleared.